src/Diagnostics.RuntimeHost/Utilities/DateTimeHelper.cs

```diff
@@ -66,10 +66,11 @@ internal static bool PrepareStartEndTimeUtc(string startTime, string endTime, ou
             DateTime currentUtcTime = GetDateTimeInUtcFormat(DateTime.UtcNow);
             bool result = true;
             errorMessage = string.Empty;
+            int kustoDelayInMinutes = -HostConstants.KustoDelayInMinutes;
 
             if (string.IsNullOrWhiteSpace(startTime) && string.IsNullOrWhiteSpace(endTime))
             {
-                endTimeUtc = currentUtcTime;
+                endTimeUtc = currentUtcTime.AddMinutes(kustoDelayInMinutes);
                 startTimeUtc = endTimeUtc.AddDays(-1);
             }
             else if (string.IsNullOrWhiteSpace(startTime))
```

Comment:
It is possible in this condition that the passed in end time is > (now - 15 minutes)